src/classlibnative/bcltype/number.cpp

```diff
@@ -9,312 +9,12 @@
 
 #include "common.h"
 #include "number.h"
-#include "bignum.h"
-#include "grisu3.h"
-#include "fp.h"
 
 typedef wchar_t wchar;
 
 #define SCALE_NAN 0x80000000
```

Comment:
Nit: These two constants can be deleted as well.

Reply:
Fixed.